jobs/release-monitoring/discovery/backup-container.yaml

```diff
@@ -0,0 +1,43 @@
+---
+- job:
+    name: release-monitoring-discovery/backup-container
+    display-name: 'Backup Container'
+    project-type: pipeline
+    concurrent: false
+    triggers:
+      - timed: '@hourly'
+    parameters:
+      - string:
+          name: 'releaseTagVar'
+          default: 'backup_version'
+          description: '[OPTIONAL] The manifest variable to be used as the current component release tag'
+          read-only: true
+      - string:
+          name: 'productVersionVar'
+          default: 'backup_image_tag'
+          description: '[REQUIRED] The manifest variable to be used as the current component product version'
+          read-only: true
```

Comment:
Are these 2 things not the wrong way round?

Reply:
Thinking about it, is there even a need for a backup_version variable? Can't see why there would be since it's just one of our operator and doesn't really operate a product with a version.

Reply:
From looking at the manifest file in the installer, it seems like the `backup_version` is being used as the release tag, and the `image_tag` is being used as the actual product version, the tag used for the image. I can switch these two if it makes more sense that way, i don't think it would be a problem.

Reply:
@mikenairn That's true. The image tag is being referenced in the [rhsso](https://github.com/integr8ly/installation/blob/master/roles/rhsso/defaults/main.yml#L57) but i don't see why two vars are needed. I think we could just use the `backup_version`. wdyt? @odra @pb82

Reply:
I'd agree, the tag name and the image tag should have the same value for a release.

Reply:
@mikenairn @pb82 Created a PR to remove the `backup_image_tag` and use the `backup_version` instead. https://github.com/integr8ly/installation/pull/459